cookieplone/filters/__init__.py

```diff
@@ -133,3 +133,9 @@ def image_prefix(registry: str) -> str:
 def as_semver(v: str) -> str:
     """Convert a PEP 440 version to SemVer."""
     return versions.convert_pep440_semver(v)
+
+
+@simple_filter
+def unscopped_package_name(v: str) -> str:
```

Comment:
Refactor to correct spelling of "unscoped" everywhere. I won't comment on every instance.

```suggestion
def unscoped_package_name(v: str) -> str:
```